smoke-tests-otel-starter/src/test/java/io/opentelemetry/smoketest/OtelSpringStarterSmokeTest.java

```diff
@@ -157,6 +166,43 @@ AutoConfigurationCustomizerProvider propagatorCustomizer() {
     }
   }
 
+  @BeforeEach
+  void setUp() {
+    if (!initialized) {
+      initialized = true;
+      assertFirstTimeSetup();
```

Comment:
Having assertions on multiple places makes difficult to know what is verified. Could you please move the assertions contained in the `setUp` method to a test method?